lib/schemas/ResourceSchema.js

```diff
@@ -151,26 +189,12 @@ module.exports = makeSchema(
         maxLength: 255
       },
       // TODO: do we need to break these all apart too? :-/
-      get: {
-        description: ResourceMethodGetSchema.schema.description,
-        $ref: ResourceMethodGetSchema.id
-      },
-      hook: {
-        description: ResourceMethodHookSchema.schema.description,
-        $ref: ResourceMethodHookSchema.id
-      },
-      list: {
-        description: ResourceMethodListSchema.schema.description,
-        $ref: ResourceMethodListSchema.id
-      },
-      search: {
-        description: ResourceMethodSearchSchema.schema.description,
-        $ref: ResourceMethodSearchSchema.id
-      },
-      create: {
-        description: ResourceMethodCreateSchema.schema.description,
-        $ref: ResourceMethodCreateSchema.id
-      },
+      get: makeSingleOrNestedMultiple(ResourceMethodGetSchema),
```

Comment:
Does multiple `get` make sense? Worried that make future ideas tough if there could be more than one get to choose from.

Reply:
Yeah, that one maybe not -- can swap that one back.